### DISCUSSION TITLE
Turn off the parenthesis on SQL Server Stored Procedure parameter autocomplete

### DISCUSSION THREAD

**Discussion by DaveBabler** [dbeaver/dbeaver#11228] [Q&A]
I have looked through the documentation.  I have Googled, Binged, checked Reddit, and even lowered myself to look through Yahoo!
I have tried messing with the preferences in the SQL PROCESSING options/preferences box.  

Is there a way to turn off the addition of the parenthesis around the automatically output parameters for stored procedures when using MS SQL SERVER?

I would also like the option to also get rid of the automatically placed colons.

The variable autocomplete is great but the parenthesis and colon drives me crazy

See picture.

Any help with figuring out how to turn this off, while still keeping the autocomplete menu and automatic variable placement after the procedure name, would be greatly appreciated.  Clearly I am far too stupid to figure this out on my own. 

![dbeaverAnnoyance](https://user-images.githubusercontent.com/36081274/106905224-0fd55c00-66ca-11eb-9d65-5e45e44f0399.png)

**Comment by LonwoLonwo** [comment]
Hello, @DaveBabler 
DBeaver - is universal Database instrument, therefore, some of our functions use the standard of the most popular databases. For example, most often in other databases, procedure parameters are stored in parentheses. 
We don't have a special setting to remove only parentheses and a colon.
I can suggest that you create a ticket for this situation on our board  - https://github.com/dbeaver/dbeaver/issues.
And we will try to figure out how best to solve this problem for SQL Server.
